new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1155 @@
+
+---> \item Apple {expect:* Apple}
+---> \item Banana {expect:* Banana}
+---> \item Orange {expect:* Orange}
+---> \item Pear {expect:* Pear}
+---> \item Lemon {expect:* Lemon}
+
+# Lesson 3.4: executing commands on visual selections
+
+** We can execute a command over only the selected part **
+
+  1. Select the sentence visually
+
+  2. Press «:» to enter command mode. The command will already be be prefixed
+     with a range: ':'<,'>'
+
+  3. Complete the command to read ':'<,'>w selection.txt' and press enter to
+     save only the selected slice to a file.
+
+---> Select this sentence.
+
+The strings "'<" and "'>" act as delimiters for a range: the range of the
+visual selection. Most commands can operate on a range.
+
+# Lesson 3.5: MODIFYING A SELECTION
+
+** Change the visually selected region using «o» **
+
+We know that we can use any motion to change the contents of our selection, but
+they work only on one end of the selection. We can toggle to the other end by
+pressing «o» when in visual mode.
+
+  1. Make a visual selection of any kind on the lines below and try modifying
+     it.
+
+  2. Press «o» and try modifying it again.
+
+---> a) Roses are red, {expect:NULL}
+---> b) Violets are blue, {expect:NULL}
+---> c) Intelligence is learned, {expect:NULL}
+---> d) Can you learn too? {expect:NULL}
+
+# Summary
+
+Visual mode is a tool that lets us fine-tune a selection of text before
+perforing an action on it.
+
+
+# Part 4: PERMANENT SETTINGS AND GETTING HELP
+
+** Enable Vim features **
+
+Vim has many more features than Vi, but most of them are disabled by default.
+To start using more features you have to create a "vimrc" file.
+
+  1. Start editing the "vimrc" file.  This depends on your system:
+    :e ~/.nvimrc     "for Unix-like systems
+    :e $VIM/_vimrc   "for Microsoft Windows
+
+  2. Now read the example "vimrc" file contents:
+    :r $VIMRUNTIME/vimrc_example.vim
+
+  3. Write the file with:
+    :w
+
+The next time you start Vim it will use syntax highlighting.  You can add all
+your preferred settings to this "vimrc" file.  For more information type  :help
+vimrc-intro
+
+** Use the on-line help system **
+
+You now know the basics of using Vim, enough for everyday text editing. We have
+only scratched the surface of what is possible even on the topics we covered in
+this tutorial. If you want to learn more about individual topics the user
+manual is the next place to go to.
+
+Vim has a comprehensive on-line help system.  To get started, try one of
+these three:
+    - press the <HELP> key (if you have one)
+    - press the <F1> key (if you have one)
+    - type
+        :help <Enter>
+
+Read the text in the help window to find out how the help works.
+Type  <Ctrl-w><Ctrl-w>   to jump from one window to another.
+Type  :q <Enter>    to close the help window.
+
+You can find help on just about any subject, by giving an argument to the
+":help" command.  Try these (don't forget pressing <Enter>):
```

Comment:
Yeah, I wasn't really happy with it either, I just copy-pasted what was already there. Good to see I'm not the only one who thinks that, I'll make is short and to the point.